srcpkgs/targetcli-fb/template

```diff
@@ -12,3 +12,6 @@ license="Apache-2.0"
 homepage="https://github.com/open-iscsi/targetcli-fb"
 distfiles="https://github.com/open-iscsi/targetcli-fb/archive/v${version}.tar.gz"
 checksum=7ebf340705b6fb407aadc437202074ca490116ef811e3b0b7306d57d6c065366
+make_dirs="
+	/etc/target 0755 root root
+	/etc/target/backup 0755 root root"
```

Comment:
This should be indented with spaces.

Commit message should be `targetcli-fb: create /etc/target...`

Reply:
Thx.

Reply:
> This should be indented with spaces.
> 
> Commit message should be `targetcli-fb: create /etc/target...`

Oh, it just failed because it has to be indent with tags.